activiti-cloud-services-runtime-bundle/activiti-cloud-services-message-events/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.activiti.cloud.rb</groupId>
+    <artifactId>activiti-cloud-services-runtime-bundle</artifactId>
+    <version>7.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>activiti-cloud-services-message-events</artifactId>
+  
+  <dependencies>
+    <dependency>
+      <groupId>org.activiti.cloud.rb</groupId>
+      <artifactId>activiti-cloud-services-events</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.activiti</groupId>
+      <artifactId>activiti-api-process-runtime-impl</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>org.activiti</groupId>
+      <artifactId>activiti-engine</artifactId>
+      <optional>true</optional>

```

Comment:
Why are `activiti-engine` and `activiti-api-process-runtime-impl` optional?

Reply:
I have intended to make them optional in order not to pull them together in starter. I have removed optional and changed the scope to provided as this module is not intended to embed Activiti engine.